assets/javascripts/admin_groups.js

```diff
@@ -366,9 +359,30 @@ function saveReorganizedGroups() {
   });
 
   if (ajaxQueries.length) {
-    $.ajax(ajaxQueries.shift());
+    handleQuery(ajaxQueries.shift());
   } else {
     handleSuccess({nothingToDo: true});
   }
   return false;
 }
+
+function handleQuery(query) {
+  const url = query.url;
+  delete query.url;
+  const success = query.success;
+  delete query.success;
+  const error = query.error;
+  delete query.error;
+  var body = new FormData();
```

Comment:
```suggestion
  const body = new FormData();
```

I think there are also some other variables which could be const or at least use let.

Reply:
This is not what const means in JavaScript. You can "write to" a const variable as much as you like (unless it is an imitable type). const just means that the variable won't be reassigned.